### PR TITLE
Port update repo cli

### PR DESCRIPTION
**Pull request by johnsonw**

### CODE REVIEW

File: chroma_core/migrations/0006_updateyumfilejob.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (johnsonw) [inline chroma_core/migrations/0006_updateyumfilejob.py]: This will change

File: iml-manager-cli/src/update_repo_file.rs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        class_name: "UpdateYumFileJob".into(),
+                        args: HostId { host_id: x.id },
+                    })
+                    .collect(),
+                message: "Update IML Agent Repo files".into(),
+            };
+
+            let stop_spinner = start_spinner("Updating Repo files...");
+
+            create_command(cmd).map(|cmd| (cmd, stop_spinner))
```
Comment (johnsonw) [inline iml-manager-cli/src/update_repo_file.rs]: I'm having an issue here... I should be able to return a future but it only lets me return `()`

File: iml-manager-cli/src/update_repo_file.rs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        .any(|y| &y.fqdn == host || &y.nodename == host)
+}
+
+#[derive(serde::Serialize)]
+pub struct HostId {
+    host_id: u32,
+}
+
+pub fn update_repo_file_cli(config: UpdateRepoFileHosts) -> () {
+    let r = hostlist_parser::parse(&config.hosts).expect("Couldn't parse hostlist expression.");
```
Comment (jgrund) [inline iml-manager-cli/src/update_repo_file.rs]: Nit, would be better to return the error here.

File: iml-manager-cli/src/update_repo_file.rs [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            if let Some((cmd, stop_spinner)) = result {
+                stop_spinner(None);
+                display_cmd_state(&cmd);
+                exit(exitcode::OK)
+            } else {
+                exit(exitcode::OK);
+            }
+        }
+        Err(e) => {
+            eprintln!("Error updating repo file: {}", e);
```
Comment (jgrund) [inline iml-manager-cli/src/update_repo_file.rs]: Errors need to stop the spinner as well.

**Comment by johnsonw** [comment]
Stratagem CLI:
stratagem_cli

**Comment by johnsonw** [comment]
update-repo-restart-service

**Comment by johnsonw** [comment]
Another update:
updates-ready

**Comment by johnsonw** [comment]
Adding servers using the cli:
add-servers

**Pull request merged**